packages/react/src/lazy.ts

```diff
@@ -0,0 +1,43 @@
+import { type ComponentType, type LazyExoticComponent, lazy as originalLazy } from 'react'
+import { noop } from './utils/noop'
+
+/**
+ * A safe wrapper around React.lazy that handles component loading failures gracefully
+ *
+ * @experimental This is experimental feature.
+ *
+ * @example
+ * ```tsx
+ * import { lazy } from '@suspensive/react'
+ *
+ * const Component = lazy(() => import('./Component'), { safe: true })
+ * ```
+ */
+export const lazy = <T extends ComponentType<any>>(
+  load: () => Promise<{ default: T }>,
+  options?: { safe: boolean }
+): LazyExoticComponent<T> & {
+  load: () => Promise<void>
+} =>
+  Object.assign(
+    originalLazy(
+      options?.safe
```

Comment:
```suggestion
  options: { safe: boolean } = {}
): LazyExoticComponent<T> & {
  load: () => Promise<void>
} =>
  Object.assign(
    originalLazy(
      options.safe
```

Adding default value can reduce optional chainings